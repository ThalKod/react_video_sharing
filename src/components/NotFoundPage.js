import React from "react";


export default function NotFoundPage(){
  return (
      <div>Error 404 !</div>
  )
};

