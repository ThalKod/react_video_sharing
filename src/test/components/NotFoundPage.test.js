import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import NotFoundPage from "../../components/NotFoundPage";

describe("NotFoundPage component", () => {
  it("should render the NotFoundPage correctly", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
