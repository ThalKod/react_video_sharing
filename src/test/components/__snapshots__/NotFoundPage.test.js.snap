// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should render the NotFoundPage correctly 1`] = `
<div>
  Error 404 !
</div>
`;
